ros_gz_sim_demos/launch/air_pressure.launch.xml

```diff
@@ -0,0 +1,15 @@
+<launch>
+  <gz_server
+    world_sdf_file="sensors.sdf"
+    use_composition="true"
+    create_own_container="true" />
```

Comment:
```suggestion
    use_composition="True"
    create_own_container="True" />
```

Reply:
> I had to use upper case for 'true' otherwise the code was not working, is this happening to you ?

That seems like a bug then. I think we should support both "True" and "true".

Reply:
I have a fix for this. I'll open a PR soon.